victor_description/urdf/victor.urdf.xacro

```diff
@@ -41,18 +45,69 @@
     </link>
 
 
-
     <!-- arm to gripper adapter -->
     <xacro:include filename="$(find victor_description)/urdf/iiwa7_to_robotiq_adapter.xacro"/>
     <xacro:iia7_to_robotiq_adapter parent="$(arg robot_name)_left_arm_flange" prefix="$(arg robot_name)_left">
-        <!--<origin xyz="0 0 0" rpy="${PI/2} 0 ${PI/4}"/> &lt;!&ndash; Offset measured with a tape measure; please confirm/fix this value &ndash;&gt;-->
         <origin xyz="0 0 0" rpy="0 0 0"/>
     </xacro:iia7_to_robotiq_adapter>
     <xacro:iia7_to_robotiq_adapter parent="$(arg robot_name)_right_arm_flange" prefix="$(arg robot_name)_right">
-        <!--<origin xyz="0 0 0" rpy="${PI/2} 0 ${PI/4}"/> &lt;!&ndash; Offset measured with a tape measure; please confirm/fix this value &ndash;&gt;-->
         <origin xyz="0 0 0" rpy="0 0 0"/>
     </xacro:iia7_to_robotiq_adapter>
 
+    <xacro:if value="$(arg use_tools)">
```

Comment:
I would put these tool macros in separate files

`<xacro:include filename="$(find victor_description)/urdf/tools/($arg tool_name).xacro">`

This makes it easy to change the robot tool without editing this `victor.urdf.xacro` file, and there does not need to be any special overwritting of the robot description on the parameter server. The same code can be synced across all accounts, with a slowly growing database of tools.